### PR TITLE
Fix Feature Info chart updating for SOS

### LINKED ISSUE #2511: SOS doesn't always update feature info when you change concept
If you use the bom.json init file, and use one of the last two catalog items, and change from Annual Average to Monthly Average, you'll see the feature info doesn't update.

### PR DESCRIPTION
**Pull request by RacingTadpole**
Fixes #2511 .

### CODE REVIEW

**Comment by RacingTadpole** [comment]
Actually I'm now seeing something a bit odd with one of the other SOS items, let me investigate.

**Comment by RacingTadpole** [comment]
OK, the problem was that now half-updated features can appear in the feature info (but only as part of getting to the fully-updated feature), so the Feature Info section needs to be able to handle that.

This actually fixes a bug I hadn't noticed before, eg. for Storage Levels, choose Daily Average and choose a sensor near Perth. Then click Annual Average, and for the date it picks, there is no such feature.  In the past, it led to weird behaviour, like being able to click anywhere and have the cross-hairs go there (because it hit an error). Now it works properly.

**Comment by RacingTadpole** [comment]
OK, this is ready for review - thanks! @kring

File: lib/Models/TableDataSource.js [hunk 1]
```diff
@@ -326,7 +326,6 @@ function setOneFeaturePerRow(dataSource, tableStructure, specialColumns) {
     // If there more entities already exist than there are in the table,
     // remove the extras.
     var entities = dataSource._entityCollection;
-    entities.suspendEvents();
```
Comment (kring) [inline lib/Models/TableDataSource.js:329]: Why was it necessary to remove these lines?  I have to imagine it makes loading a CSV much slower i some scenarios.

Comment (kring) [reply lib/Models/TableDataSource.js:329]: Oh nevermind, I see now you just moved the calls to the calling function

**Pull request merged**